lib/logger.js

```diff
@@ -21,7 +21,7 @@ const kMeta = Symbol('meta')
 const kIsLoggingBackedOff = Symbol.for('isLoggingBackedOff')
 const kFlusher = Symbol('flusher')
 const kAttempts = Symbol('attempts')
-const kMaxAttempts = Symbol('maxAttempts')
+const kMaxAttempts = Symbol.for('maxAttempts')
```

Comment:
feels like we should avoid the inconsistency and just split out `./symbols.js` at this point, so we can access these under test. nothing to do here just stood out in the diff.

Reply:
Yes, this goes right along with a discussion we had about making sure nothing can be missed in testing